pkg/activator/throttler.go

```diff
@@ -133,6 +147,20 @@ func (t *Throttler) Try(rev RevisionID, function func()) error {
 	return nil
 }
 
+func (t *Throttler) activatorEndpointsUpdated(newObj interface{}) {
+	endpoints := newObj.(*corev1.Endpoints)
+	count := resources.ReadyAddressCount(endpoints)
+	atomic.StoreInt32(&t.activatorCount, int32(count))
```

Comment:
I think this should kick off a "global resync" and call `forceUpdateCapacity` on all breakers.

Reply:
Thats a good point. We will update this

Reply:
On second thought if we do take global resync approach it doesn't provide any additional benefits. 
If revision instances scale up/down then the endpoints resource is updated as well. Callback function on endpoints informer triggers `UpdateCapacity` function which will use current value of activator count. What problem / issue would global resync approach solve that would be missed in current approach?

Reply:
If the activator scales down, the UpdateEndpoints on the indivual breakers is not done or am I missing something?

Reply:
Note: I don‘t mean an actual global resync via the activator but rather iterate through all breakers here and call forceUpdateCapacitx for them.